src/Entity/Task/RecurrenceRule.php

```diff
@@ -21,31 +21,31 @@
  *   collectionOperations={
  *     "get"={
  *       "method"="GET",
- *       "security"="is_granted('ROLE_ADMIN')"
+ *       "security"="is_granted('ROLE_ADMIN') or is_granted('ROLE_DISPATCHER')"
```

Comment:
Should we maybe specify only `is_granted('ROLE_DISPATCHER')` here? And rely on role hierarchy to grant admins the same permissions: https://github.com/coopcycle/coopcycle-web/blob/master/app/config/security.yml#L19